doc/classes/TileMap.xml

```diff
@@ -227,12 +227,12 @@
 			<argument index="1" name="coords" type="Vector2i" />
 			<argument index="2" name="source_id" type="int" default="-1" />
 			<argument index="3" name="atlas_coords" type="Vector2i" default="Vector2i(-1, -1)" />
-			<argument index="4" name="alternative_tile" type="int" default="-1" />
+			<argument index="4" name="alternative_tile" type="int" default="0" />
 			<description>
 				Sets the tile indentifiers for the cell on layer [code]layer[/code] at coordinates [code]coords[/code]. Each tile of the [TileSet] is identified using three parts:
 				- The source identifier [code]source_id[/code] identifies a [TileSetSource] identifier. See [method TileSet.set_source_id],
 				- The atlas coordinates identifier [code]atlas_coords[/code] identifies a tile coordinates in the atlas (if the source is a [TileSetAtlasSource]. For [TileSetScenesCollectionSource] it should be 0),
-				- The alternative tile identifier [code]alternative_tile[/code] identifies a tile alternative the source is a [TileSetAtlasSource], and the scene for a [TileSetScenesCollectionSource].
+				- The alternative tile identifier [code]alternative_tile[/code] identifies a tile alternative the source is a [TileSetAtlasSource], and the scene for a [TileSetScenesCollectionSource]. Passing -1 will clear the tile.
```

Comment:
I don't think this precision should be here. You have to set all three args to -1, Vector2i(-1,-1) and -1 to clear a tile, not only the last one.